src/elements/Segment/Segment.js

```diff
@@ -2,10 +2,22 @@ import React, {Component, PropTypes} from 'react';
 import classNames from 'classnames';
 import META from 'src/utils/Meta';
 
+/**
+ * A segment is used to create a grouping of related content.
+ */
 export default class Segment extends Component {
   static propTypes = {
+    /**
+     * Segment tag body content.
+     */
     children: PropTypes.node,
+    /**
+     * Class name for custom styling.
+     */
```

Comment:
I'm torn on this particular doc block.  Partly because I think it is common knowledge.  Mostly because it seems if we add this one to `className` then we should add the same to all `className` props for all components.  If we did this, would suggest doing it in `ComponentProps.js`, where the prop types doc section is generated.  Then all components can have the same default doc string added in one place.  We could do this for `className` `style` `children`, etc.  Then, save component level doc strings for component specific things that folks aren't expected to know, like `heading`.

Thoughts?